skyvern/webeye/actions/handler.py

```diff
@@ -316,6 +317,17 @@ async def handle_select_option_action(
             select_action = SelectOptionAction(element_id=select_element_id, option=action.option)
             return await handle_select_option_action(select_action, page, scraped_page, task, step)
 
+        # handle the select action on <label> of checkbox/radio
+        checkbox_element_id = get_checkbox_id_in_label_children(scraped_page, action.element_id)
```

Comment:
before we check children of label, can we check if the which element the label is `for`?

If it's not a checkbox, then we can try looking at its children 🤔

Reply:
how about doing it in another ticket?
since it's a more complicated situation.
Like we might get the label `for`, bonding to the <select>  or <input> in the select action.
both should be handled.

Reply:
some elements bond with the `<label>` might be invisible on the page. 
go to https://github.com/Skyvern-AI/skyvern/pull/273 for more details.
and the select and check should be handled separately, so it's a little complicated.